addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -44,6 +44,10 @@ export class Attachment extends FileModelMixin(Record) {
         return `${this.create_date.monthLong}, ${this.create_date.year}`;
     }
 
+    get uploading() {
+        return this.id < 0;
+    }
+
```

Comment:
Now that the uploading state is deduced from id and properly part of the attachment model, we probably want to remove it from the `attachment_upload_service`.

Reply:
Yes, this can be cleaned in master, just to avoid diffing in stable too much, and also this can be considered as breaking change if someone was using it somehow.